examples/07_cloudflare/src/lib/hono.ts

```diff
@@ -1,12 +1,12 @@
-import { getHonoContext as getHonoContextFromWaku } from 'waku/unstable_hono';
+import { getHonoContext as _getHonoContext } from 'waku/unstable_hono';
```

Comment:
It's our coding style to use `_` prefix for "unused values", so we want to avoid using `_` for other meanings.